### DISCUSSION TITLE
/graph API is missing with cortex endpoint.

### DISCUSSION THREAD

**Discussion by uvaisibrahim** [cortexproject/cortex#4058] [General]
We have Thanos as backend to the Cortex frontend. While hitting cortex endpoint with /graph, we were getting the Thanos web console. But after upgraded to 1.6.0 from 1.0.1, the /graph uri is not working, giving 404. Can somebody please give an eye opening reply.

**Comment by pstibrany** [comment]
This was discussed at https://github.com/cortexproject/cortex/issues/2742, with no solution.

Supporting /graph was not really intentional in the first place.
